russian/buildings-dep/buildings-dep.md

```diff
@@ -1,4 +1,8 @@
 Вы дозвонились в ЖЭК, хотя и боялись, что вас не поймут. Объяснили
 ситуацию и попросили помочь. Вас внимательно выслушали, нос казали
 что все сейчас заняты, так как Ваш звонок такой не первый за день и
-продеться подождать.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.